modelscope/pipelines/cv/image_matching_fast_pipeline.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+from typing import Any, Dict, List, Union
+
+import cv2
+import numpy as np
+import PIL
+import torch
+
+from modelscope.metainfo import Pipelines
+from modelscope.outputs import OutputKeys
+from modelscope.pipelines.base import Input, Model, Pipeline
+from modelscope.pipelines.builder import PIPELINES
+from modelscope.preprocessors import LoadImage
+from modelscope.utils.constant import Tasks
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+@PIPELINES.register_module(
+    Tasks.image_matching, module_name=Pipelines.image_matching_fast)
+class ImageMatchingFastPipeline(Pipeline):
+
```

Comment:
add pipeline doc str, refer to https://github.com/modelscope/modelscope/blob/master/modelscope/pipelines/cv/card_detection_correction_pipeline.py#L34C1-L34C1

Reply:
Done.